src/linter/tests/reporters/console.test.mjs

```diff
@@ -1,26 +1,36 @@
 import { describe, it } from 'node:test';
-import console from '../../reporters/console.mjs';
 import assert from 'node:assert';
+import reporter from '../../reporters/console.mjs';
 import { errorIssue, infoIssue, warnIssue } from '../fixtures/issues.mjs';
 
-describe('console', () => {
-  it('should write to stdout with the correct colors based on the issue level', t => {
-    t.mock.method(process.stdout, 'write');
-
-    console(infoIssue);
-    console(warnIssue);
-    console(errorIssue);
+const testCases = [
+  {
+    issue: infoIssue,
+    method: 'info',
+    expected: '\x1B[90mThis is a INFO issue at doc/api/test.md\x1B[39m',
+  },
+  {
+    issue: warnIssue,
+    method: 'warn',
+    expected: '\x1B[33mThis is a WARN issue at doc/api/test.md (1:1)\x1B[39m',
```

Comment:
Can we use snapshot ?

Reply:
I don't think we use snapshot elsewhere, so maybe you can open a follow up to change them all?